R/utils_preprocess_register.R

```diff
@@ -11,6 +11,50 @@ register_clear_cache <- function() {
   unlink(path, recursive = TRUE)
 }
 
+#' Function for adding clickable links to the paper for each entry in the register table.
+#' 
+#' @param register_table The register table
+#' @param register The register from the register.csv file
+#' @return The adjusted register table
+add_paper_links <- function(register_table, register){
+  list_hyperlinks <- c()
+
+  # Looping over the entries in the register
+  for (i in seq_len(nrow(register))) {
+    # Retrieving the link to the paper 
+    config_yml <- get_codecheck_yml(register[i, ]$Repo)
+    paper_link <- config_yml[["paper"]][["reference"]]
+    paper_title <- config_yml[["paper"]][["title"]]
+
+    # Removing new lines from paper title and link
```

Comment:
We can leave this as is now, but since I'm not afraid of dependencies for the current setup, we could also use the `stringr` package for this kind of task: https://stringr.tidyverse.org/reference/str_trim.html IMO that would be less error-prone and also more readable (for the regex-challenged like me).

Nothing to change now unless you're intrigued, but keep in mind for the future.